### PR TITLE
Register member

### PR DESCRIPTION
**Pull request by DinolanNaidoo**

### CODE REVIEW

File: skietbaan_fe/src/components/NavbarMenu.js [hunk 1]
```diff
@@ -4,19 +4,19 @@ import '../scss/navbar.css';
...
-import createRed from '../components/navbar-icons/create-red.png';
-import createGray from '../components/navbar-icons/create-grey.png';
-import leaderboardRed from '../components/navbar-icons/leaderboard-red.png';
-import leaderboardGray from '../components/navbar-icons/leaderboard-grey.png';
-import notificationsRed from '../components/navbar-icons/notifications-red.png';
-import notificationsGray from '../components/navbar-icons/notifications-grey.png';
-import scorecaptureRed from '../components/navbar-icons/add-score-red.png';
-import scorecaptureGray from '../components/navbar-icons/add-score-grey.png';
-import moreGray from '../components/navbar-icons/more-grey.png';
+import profilered from '../components/navbar-icons/profile-red.png';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenu.js]: This was correct

File: skietbaan_fe/src/components/NavbarMenuUser.js [hunk 2]
```diff
@@ -2,17 +2,17 @@ import React, { Component } from 'react';
...
-import docsRed from '../components/navbar-icons/docs-red.png';
-import docsGray from '../components/navbar-icons/docs-grey.png';
-import leaderboardRed from '../components/navbar-icons/leaderboard-red.png';
-import leaderboardGray from '../components/navbar-icons/leaderboard-grey.png';
-import notificationsRed from '../components/navbar-icons/notifications-red.png';
-import notificationsGray from '../components/navbar-icons/notifications-grey.png';
-import scorecaptureRed from '../components/navbar-icons/add-score-red.png';
-import scorecaptureGray from '../components/navbar-icons/add-score-grey.png';
-import moreGray from '../components/navbar-icons/more-grey.png';
+import profilered from '../components/navbar-icons/profile-red.png';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenuUser.js]: this was correct

File: skietbaan_fe/src/components/NavbarMenu.js (hunk 1 above)
Comment (FredJEhlers) [reply skietbaan_fe/src/components/NavbarMenu.js]: if you create an icons constant object then you can have one import in here like 
import NAV_BAR_ICONS from 'constantsFile'
then just  `src={ NAV_BAR_ICONS.PROFILE_GRAY }`
just use real names, no docsgray more like DOCUMENT_GRAY

File: skietbaan_fe/src/components/NavbarMenuUser.js (hunk 2 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenuUser.js]: changed

File: skietbaan_fe/src/components/NavbarMenu.js (hunk 1 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenu.js]: okay done

File: skietbaan_fe/src/actions/types.js [hunk 3]
```diff
@@ -1,3 +1,33 @@
+import profilered from '../components/navbar-icons/profile-red.png';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/actions/types.js]: camelCase

File: skietbaan_fe/src/components/CreatePage.js [hunk 4]
```diff
@@ -30,14 +31,17 @@ class CreatePage extends Component {
...
+		window.location = "/viewMembers";
 	}
 
 	ViewGroups() {
 		window.location = "#";
 	}
 
 	render() {
+		if(!getCookie("token")){
+            window.location = "/register-page";
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/CreatePage.js]: Others are camelCases change this to match please

File: skietbaan_fe/src/components/RegisterMember.js [hunk 5]
```diff
@@ -149,13 +150,18 @@ class App extends Component {
   }
 
   render() {
+    if(!getCookie("token")){
+      window.location = "/register-page";
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/RegisterMember.js]: Same note from above

File: skietbaan_fe/src/components/ViewMembers.js [hunk 6]
```diff
@@ -3,7 +3,8 @@ import '../components/ViewMembers.css';
 import { Table } from 'react-bootstrap';
 import Collapsible from 'react-collapsible';
 import { BASE_URL } from '../actions/types.js';
-
+import membericon from '../components/assets/membership-icon.png';
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.js]: camelCase please

File: skietbaan_fe/src/components/ViewMembers.js [hunk 7]
```diff
@@ -94,6 +95,9 @@ class ViewMembers extends Component {
     }
 
     render() {
+        if(!getCookie("token")){
+            window.location = "/register-page";
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ViewMembers.js]: Same from previous comments

File: skietbaan_fe/src/actions/types.js (hunk 3 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/actions/types.js]: done

File: skietbaan_fe/src/components/CreatePage.js (hunk 4 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/CreatePage.js]: okay

File: skietbaan_fe/src/components/ViewMembers.js (hunk 6 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.js]: done

File: skietbaan_fe/src/components/ViewMembers.js (hunk 7 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/ViewMembers.js]: done

**Pull request merged**